prototypes/anim_polygons_data/Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM debian:bullseye-backports
+
+WORKDIR /root
+
+RUN apt-get update && apt-get upgrade -y
+RUN apt-get install -y --no-install-recommends \
+    build-essential \
+    python3 curl git \
+    ca-certificates \
+    python3-pip xauth \
+    libopenexr-dev \
+    bzip2 \
+	zlib1g-dev \
```

Comment:
Is there a particular reason why following packages are indented in a different way?

Reply:
There's no reason, in fact vim hid the indentation from me.
![image](https://user-images.githubusercontent.com/63677127/168785954-21d28a7e-24c5-4890-9b28-e2c6af07b0e8.png)